docs/configuration/events/weather-events.rst

```diff
@@ -0,0 +1,66 @@
+Stops
+=====================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+
+Prerequisites
+-------------------------------------
+
+This pages assumes the following:
+
+1. You have a working scanner.
+2. You read and understood the :ref:`events_dts` page.
+3. You are using the latest version of PokeAlarm.
+
+
+Description
+-------------------------------------
+
+A **Weather Event** represents a change in the weather for a specific s2 cell.
+
+
+Available DTS
+-------------------------------------
+
+
+General
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+=============== ===========================================================================
+DTS             Description
+=============== ===========================================================================
+s2_cell_id      The id of the s2 cell. Unique per cell.
+weather_id      The id of the changed weather condition.
+weather_emoji   The emoji representing the changed weather condition.
+alert_id        The id that represents the severity of the weather.
```

Comment:
What are the possible values for this? Would a better name be perhaps `severity_id`?